.travis.yml

```diff
@@ -1,9 +1,27 @@
 language: objective-c
-osx_image: xcode6.4
+osx_image: xcode7.2
+sudo: false
+xcode_project: SocketRocket.xcodeproj
+env:
```

Comment:
ahhhh this makes it so we don't have to add new `xcodebuild` lines to the script below. Dig the concept